frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -129,6 +130,11 @@ private boolean isManagementReq(String[] segments) {
                 || endpointMap.containsKey(segments[1]);
     }
 
+    private boolean isModeExplicit() {
+        ConfigManager configManager = ConfigManager.getInstance();
```

Comment:
Nit: `ConfigManager` is singleton so we can fetch this only once and save it in a class variable and just do `return configManager.getModelControlMode()` here.